### PR TITLE
[ISSUE-460] - Font Size Audit

### LINKED ISSUE #460: Audit default fonts & standardize size  if applicable.
The goal of this ticket is to audit gesso-wp's font system as it stands and establish a standard (slightly more trim) for pre-defined sizes and families. 

AC:
* Reduce the amount of pre-defined fonts in the theme.json to a conventional baseline of seven different fonts OOTB
* Rename them to follow the following convention: 

>xxx-small (14px)
>xx-small
>x-small
>Small
>Medium
>Large
>x-large
>xx-large
>xxx-large

The reason for changing our predefined fonts to the label and amount above is two fold:

1. The names we define in theme.json are human readable in the CMS when people are managing their selected font site. Right now they just display numbers which can be a little confusing. The seven defined names above follow the WP Core convention and are a bit more human readable. 
2. We generally have more defined fonts than we have in a theme design. We know we will have a standard of 6 sizes for headers, as well as at least 1 additional size, leading to a baseline of 7 that we can add to as designs need more. 

To determine what you feel is the best baseline for those sizes, please take a look at the WP 2024 theme to compare our current font sizes and families against what is being implemented by the core team: https://wordpress.org/themes/twentytwentyfour/ Their theme.json will provide a strong foundation to compare our slew of font size declarations against. You can also take a look at some of our more modern figma examples for what our in house convenstions have been as of late:

https://www.figma.com/design/53pTSftpSI6AUqaMqpPojN/Accelerator-(Gates-Foundation)?node-id=4679-6686&t=w8nu8xpgA0GT0eyo-0

Please reference the standard convention for min/max values for the fluid fonts within the 2024 theme. Their size is generally matching the max value. Please also do not apply a fluid font size that ranges below 14px for accessibility reasons.

### PR DESCRIPTION
**Pull request by christopherhodges**
**Resolves #460** 
- Removed gesso font-sizes
- Replaced with new settings ranging from xx small - xx large
- Total of 7 fontSize declarations

### CODE REVIEW

File: theme.json [hunk 1]
```diff
@@ -188,127 +188,61 @@
...
+          "fluid": {
+            "min": "22px",
+            "max": "24px"
+          }
         },
         {
-          "name": "9",
-          "slug": "9",
+          "name": "X Large",
+          "slug": "xx-small",
```
Comment (tropicandid) [inline theme.json]: I believe this is the wrong slug here. This should be x-large in this new convention.

File: theme.json [hunk 2]
```diff
@@ -188,127 +188,61 @@
...
+          "fluid": {
+            "min": "24px",
+            "max": "28px"
+          }
         },
         {
-          "name": "12",
-          "slug": "12",
+          "name": "XX Large",
+          "slug": "xx-small",
```
Comment (tropicandid) [inline theme.json]: Please correct the slug here.

**Review by tropicandid — CHANGES_REQUESTED** [review]
@christopherhodges There are two small slug changes, but One thing that is a late breaking request from FEDs. The request was to start with 9 fonts (6 header and 3 body) would you mind adding two additional font sizes and adjusting the slugs and sizes accordingly? I updated the associated issue for this. Other than those two things this is looking good! Sorry for the last minute change to AC

**Comment by christopherhodges** [comment]
@tropicandid this should be good to go! I set the largest font-size to 44px. I tried to mimic what twentytwentyfour uses but in pixel units instead of rems.

File: theme.json (hunk 1 above)
Comment (tropicandid) [reply theme.json]: This has been updated.

File: theme.json (hunk 2 above)
Comment (tropicandid) [reply theme.json]: This has been resolved. TY!

**Review by tropicandid — APPROVED** [review]

**Pull request merged**